apps/mull-ui/src/app/components/profile-header/profile-header.tsx

```diff
@@ -146,30 +147,41 @@ export const ProfileHeader = ({
         <img className="user-profile-picture" src={avatarUrl(user)} alt="user" />
         <div
           className={
-            isCurrentUser ? 'profile-side-container for-current-user' : 'profile-side-container'
+            isCurrentUser
+              ? 'profile-side-container for-current-user'
+              : 'profile-side-container for-other-user'
           }
         >
-          <button className="profile-stats" data-testid="portfolioCount">
+          <p
+            className={isCurrentUser ? 'profile-stats' : 'profile-stats for-other-user'}
```

Comment:
I find this is less repetition and easier read
```suggestion
            className={`profile-stats ${isCurrentUser ? 'for-other-user': ''}`}
```

Reply:
Since a couple of these have the same class name, you could even extract the classname into a variable